node/frontend/main.js

```diff
@@ -38,18 +25,26 @@ const corsOptions = {
 
 app.use(cors(corsOptions))
 
+const tracer = opentelemetry.trace.getTracer(
+  'default'
+);
+
 app.get('/greeting', async (req, res) => {
-  beeline.addContext({ name: 'Greetings' });
-  const greetingSpan = beeline.startSpan({ name: 'Preparing Greeting' });
-  beeline.addTraceContext({ name: 'Greetings' });
-  beeline.finishSpan(greetingSpan);
-  const nameSpan = beeline.startSpan({ name: '✨ call /name ✨' });
+  const greetingSpan = tracer.startSpan('✨ preparing greeting ✨');
+  greetingSpan.end()
+
+  const nameSpan = tracer.startSpan('✨ call /name ✨');
   const name = await getName(nameUrl);
-  beeline.finishSpan(nameSpan);
-  const messageSpan = beeline.startSpan({ name: '✨ call /message ✨' });
+  nameSpan.setAttribute("app.user_name", name);
+  nameSpan.end()
+
+  const messageSpan = tracer.startSpan('✨ call /message ✨');

```

Comment:
Do we want to record the message we get here as an attribute as well?

Reply:
✅